From b371c4270f4a6a0ca96182bd5cc7ea5016c368ce Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Mon, 20 Aug 2018 15:50:07 +0000
Subject: [PATCH 319/462] MANTIS 51444 BhsServer writes monitoring data to
 dedicated OPC UA servers - Handshake via Dispatch Data Acknowledgement

Fixed typo in property JarGuava

git-svn-id: svn://localhost/SelfServiceBuild/trunk@554 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 32687d5..20dfd57 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -344,7 +344,7 @@
 <property name="JarXStream"               value="${DIR_GLOBALS}/ThirdParty/xstream/1.4.6/xstream-1.4.6.jar"/>
 
 <!-- guava -->
-<property name="JarGuava"                   value="${DIR_GLOBALS}/guava/23.3-android/guava-23.3-android.jar"/>
+<property name="JarGuava"                   value="${DIR_GLOBALS}/ThirdParty/guava/23.3-android/guava-23.3-android.jar"/>
 
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
 
-- 
2.41.0.windows.1

